cdap-cli/src/main/java/co/cask/cdap/cli/util/AbstractAuthCommand.java

```diff
@@ -72,29 +76,38 @@ public void execute(Arguments arguments, PrintStream printStream) throws Excepti
   public abstract void perform(Arguments arguments, PrintStream printStream) throws Exception;
 
   protected ServiceId parseServiceId(Arguments arguments) {
```

Comment:
not your change, but it doesn't seem like ``parseServiceId`` should be in AbstractAuthCommand.